<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QServer</class>
 <widget class="QGroupBox" name="QServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>193</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <property name="title">
   <string>Server</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="2" colspan="2">
    <widget class="QLabel" name="lx_port">
     <property name="text">
      <string>port</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="3">
    <layout class="QHBoxLayout" name="hl_buttons">
     <item>
      <widget class="QPushButton" name="bt_apply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_discard">
       <property name="text">
        <string>Discard</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="4">
    <widget class="QSpinBox" name="sb_port">
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>port</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>80</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLabel" name="lx_ip">
     <property name="text">
      <string>IP address</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLineEdit" name="le_ip">
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>none</string>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="lx_station_id">
     <property name="text">
      <string>Station ID</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="3" colspan="2">
    <widget class="QPushButton" name="bt_send_heartbeat">
     <property name="text">
      <string>Send heartbeat now</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QLabel" name="lb_next_heartbeat">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>? s</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="lx_next_heartbeat">
     <property name="text">
      <string>Next heartbeat in</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLineEdit" name="le_station_id">
     <property name="text">
      <string>none</string>
     </property>
     <property name="maxLength">
      <number>4</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_station_id</tabstop>
  <tabstop>le_ip</tabstop>
  <tabstop>sb_port</tabstop>
  <tabstop>bt_discard</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
